### ISSUE TITLE
Not just show that there is an issue..

### ISSUE THREAD

**Issue by AlminCode** [climatepatch/formation#19]
So I see a lot of people reporting that we should make something that SHOWS that we have an issue which is ok but I think we also should provide solutions. 

Solutions could be images/videos/studies which show how we can make the air cleaner.

Ex. Tesla batteries, wind/solar farms etc. Just to show people that it is possible and that there are alternatives.

**Comment by indigocarmen** [comment]
Can I recommend you go check out the Podcast Solution? This would be a great feature of that. What I personally was envisioning for that Solution was that each week/month we/someone present a piece of technology helping to make the environment better. As an example; the recent wind mills that can power homes :http://www.wionews.com/south-asia/indian-start-up-lets-you-power-your-home-for-life-for-the-price-of-a-high-end-smartphone-7906/

**Comment by Slimfit** [comment]
I agree this would be implemented in either the gitbooks idea #9 or as a podcast/youtube channel. We dont have a dedicated thread for those ideas, so lets start it here.

**Comment by AlminCode** [comment]
Yes, Seems like a good one. So maybe weekly podcast or even newsletter with tips & tricks on how to make the earth "greener" with some easy tips everyone could follow?

**Comment by indigocarmen** [comment]
Running with this idea we could go to Inside.com and propose a Climate Change newsletter? I am pretty sure there is enough in climate change happening each week to warrant one?
